database/certificate.go

```diff
@@ -640,4 +676,9 @@ var allCertificateColumns = []string{
 	"permitted_names",
```

Comment:
`"is_name_constrained"` and `"permitted_names"` need to be removed from `allCertificateColumns`.